### PR TITLE
Corruption Level Created

### LINKED ISSUE #73: Corruption
Create a player stat that increases when an enemy dies

### PR DESCRIPTION
**Pull request by Galaxy25**
Closes #73 
One down

### CODE REVIEW

File: Assets/Scripts/Player/PlayerMovement.cs [hunk 1]
```diff
@@ -6,7 +6,6 @@
 public class PlayerMovement : MonoBehaviour
 {
     public float Speed;
-
```
Comment (BrandonPacewic) [inline Assets/Scripts/Player/PlayerMovement.cs:9]: Any particular reason why this line was deleted?

Comment (Galaxy25) [reply Assets/Scripts/Player/PlayerMovement.cs:9]: It looked off

Comment (BrandonPacewic) [reply Assets/Scripts/Player/PlayerMovement.cs:9]: I want to argue that it should stay as it separates the public and private variables. Unless we don't do this in other files?

Comment (Galaxy25) [reply Assets/Scripts/Player/PlayerMovement.cs:9]: Thats find with me

File: Assets/Scripts/Player/PlayerMovement.cs [hunk 2]
```diff
@@ -6,7 +6,6 @@
 public class PlayerMovement : MonoBehaviour
 {
     public float Speed;
```
Comment (BrandonPacewic) [inline Assets/Scripts/Player/PlayerMovement.cs:8]:
```suggestion
    public float Speed;

```

**Review by BrandonPacewic — APPROVED** [review]
One down indeed!

**Pull request merged**